spinta/backends/postgresql/types/geometry/helpers.py

```diff
@@ -27,10 +28,19 @@ def get_osm_link(value: WKBElement, srid: Optional[int]) -> Optional[str]:
             crs_to=CRS(f'EPSG:{WGS84}'),
         )
         shape = transform(transformer.transform, shape)
-
     centroid = shape.centroid
     lat, lon = centroid.x, centroid.y
     params = urlencode({'mlat': lat, 'mlon': lon})
+    if srid and srid != WGS84:
+        if 'bbox' in transformer.to_json_dict():
+            south_lat = transformer.to_json_dict().get('bbox')['south_latitude']
+            north_lat = transformer.to_json_dict().get('bbox')['north_latitude']
+            east_lon = transformer.to_json_dict().get('bbox')['east_longitude']
+            west_lon = transformer.to_json_dict().get('bbox')['west_longitude']
+            if not south_lat < lat < north_lat:
+                raise LatitudeOutOfRange
```

Comment:
Instead of just raising bare exception, pass property, that has the error. And the check should be done in `check` command. We don't want to check for errors, when reading data, we check errors, when writing data.